manuscript/manuscript.md

```diff
@@ -76,98 +76,122 @@ organization:
-[@White2013; @Goodman2014], data storage has received comparatively less
-attention.  Proper storage is a prerequisite to sharing, and indeed inadequate
-storage contributes to the phenomenon of data decay or "data entropy", where data,
-whether publicly shared or not, becomes less accessible through time
-[@Pepe2014; @Vines2014; @Michener2012; @Michener1997]. Best practices for data
-storage often begin and end with, "use a community standard repository."  This
-is good advice, especially considering your data is most likely to be reused if it is available on a community site. Community repositories can also provide guidance for best practices. As an example, if you are archiving sequencing data, it's best to consider a National Center for Biotechnology Information (NCBI) repository such as GenBank, or at least adopting their specific practices. However, data storage policies are highly variable between
-repositories [@Marcial2010]. A data management plan utilizing best practices
-across all stages of the data life cycle will facilitate transition from local
-storage to repository [@Michener2015]. Similarly, it can facilitate transition
-from repository to repository if funding runs out or needs change. Good storage
-practices are important even (or especially) in cases where data may not fit
-with an existing repository, where only derived data products (versus raw data)
-are suitable for archiving, or in the case where an existing repository may have
-lax standards.
+[@Wolkovich2012; @Roche2014] and the best practices to do so
+[@White2013; @Goodman2014], data storage has received comparatively
+less attention. Proper storage is a prerequisite to sharing, and
+indeed inadequate storage contributes to the phenomenon of data decay
+or to "data entropy", where data, whether publicly shared or not,
+becomes less accessible through time
+[@Pepe2014; @Vines2014; @Michener2012; @Michener1997]. Best practices
+for data storage often begin and end with, "deposit your data in a
+community standard repository." This is good advice, especially
+considering your data is most likely to be reused if it is available
+on a community site. Community repositories can also provide guidance
+for best practices. As an example, if you are archiving sequencing
+data, a repository such as those run by the National Center for
+Biotechnology Information (NCBI) (e.g. GenBank) provides not only a
+location for data archival, but also encourages a set of practices
+related to consistent data formatting and the inclusion of appropriate
+metadata. However, data storage policies are highly variable between
+repositories [@Marcial2010]. A data management plan utilizing best
+practices across all stages of the data life cycle will facilitate
+transition from local storage to repository [@Michener2015].
+Similarly, having such a plan can facilitate transition from
+repository to repository if funding runs out or needs change. Good
```

Comment:
This sentence confused me a little on first reading. Can we change "needs" to "requirements"?